draft-irtf-panrg-path-properties.md

```diff
@@ -195,6 +197,8 @@ It is also possible for an entity to set target properties which it cannot (dire
 For example, this can apply to security-related target properties and path selection, such as allowing or disallowing sending flows over paths that involve specific networks or nodes to enforce traffic policies or mandating that all enterprise traffic goes through a specific firewall.
 
 Care needs to be taken when selecting paths based on observed path properties, as path properties that were previously measured may not be helpful in predicting current or future path properties and such path selection may lead to unintended feedback loops. Also, there may be trade-offs between path properties (e.g., One-Way Delay and Link Capacity), and entities may influence these trade-offs with their choices.
+Finally, path selection may impact fairness, since the used paths have different properties depending on the path selection strategy or the entity's access to diverse paths and observed path properties.
+Also, if target properties depend on the application used, path selection may allow network operators to re-prioritize traffic of certain applications.
```

Comment:
I'm fine with moving the traffic prioritization part to the security considerations section.

Regarding the updated text, I would replace "path property measurements" with "experienced path properties", since not only the measurements but also the path properties themselves change.
I tried to rephrase what I meant with path diversity and observed properties in a separate sentence below, but I'm also okay with not adding this sentence.

```suggestion
Finally, path selection may impact fairness.
For example, if multiple entities concurrently attempt to meet their target properties using the same network resources, one entity's choices may influence the experienced path properties of another entity.
Or if some entities have access to a more diverse set of paths or observe more path properties than others, they may more easily be able to achieve their target property.
```

Reply:
> Regarding the updated text, I would replace "path property measurements" with "experienced path properties", since not only the measurements but also the path properties themselves change.

I agree it's not just about the measurements, but about the actual conditions on the path. Reading our definition of path property, I'm not sure if "experienced path properties" really works (can a trait be experienced?). How about simply writing "conditions on the path as experienced by flows of another entity", or more directly "network resources that are available for other entities to meet their target"?

> I tried to rephrase what I meant with path diversity and observed properties in a separate sentence below, but I'm also okay with not adding this sentence.

I see where you're going with this now, and I'm not sure about the definition of "Fairness" here. In the first part, I understood fairness in the classical sense of flows sharing a bottleneck, where it's considered unfair if one flow getting better outcomes than others. But if a flow gets better outcomes because it's using a different path because its entity has better paths available, I'm not sure if that's still captured by any widely accepted definition of fairness. Similarly, if the flow gets better outcomes because it's using, say, "better" measurement technologies, or if it's traversing specific infrastructure that is somehow aware of the flow's needs, is that unfair? I don't think I want to go into net neutrality in this document. :slightly_smiling_face: 
So I would rather delete this last sentence.

Reply:
> How about simply writing "conditions on the path as experienced by flows of another entity", or more directly "network resources that are available for other entities to meet their target"?

I like the first suggestion. See my suggestion below.

>  I don't think I want to go into net neutrality in this document. :slightly_smiling_face:

Good point, let's not wake a sleeping lion :wink:

```suggestion
Finally, path selection may impact fairness.
For example, if multiple entities concurrently attempt to meet their target properties using the same network resources, one entity's choices may influence the conditions on the path as experienced by flows of another entity.
```